### PR TITLE
Create chat-with.sx9.json

### PR DESCRIPTION
**Pull request by satr14washere**
trying to create an email lolz hopefully I can do this

### CODE REVIEW

File: domains/chat-with.sx9.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{ 
+  "description": "SX's Email", 
+  "owner": { 
+    "username": "SX-9", 
+    "twitter": "SX_Discord",
+    "email": "sx-91@outlook.com",
+  },
+  "record": { 
+    "CNAME": ["ns1.epizy.com", "ns2.epizy.com"]
```
Comment (MaskDuck) [inline domains/chat-with.sx9.json:9]: one cname only

**Review by MaskDuck — DISMISSED** [review]

File: domains/chat-with.sx9.json (hunk 1 above)
Comment (SX-9) [reply domains/chat-with.sx9.json:9]: > 

> they are name servers

Comment (MaskDuck) [reply domains/chat-with.sx9.json]: name servers arent allowed iirc

lemme ask phenax

**Comment by SX-9** [comment]
k

**Comment by MaskDuck** [comment]
nvm
image

@SX-9 cname is not allowed

**Comment by SX-9** [comment]
k

**Pull request closed without merging**